module/chunks/chunkVerifier_test.go

```diff
@@ -346,3 +396,34 @@ func (vm *vmMock) Run(ctx fvm.Context, proc fvm.Procedure, led state.View, progr
 
 	return nil
 }
+
+type vmSystemOkMock struct{}
+
+func (vm *vmSystemOkMock) Run(ctx fvm.Context, proc fvm.Procedure, led state.View, programs *programs.Programs) error {
+	tx, ok := proc.(*fvm.TransactionProcedure)
+	if !ok {
+		return fmt.Errorf("invokable is not a transaction")
+	}
+
+	tx.ServiceEvents = []flow.Event{epochSetupEvent}
+
+	_, _ = led.Get("00", "", "")
+	_, _ = led.Get("05", "", "")
+	_ = led.Set("05", "", "", []byte{'B'})
+	tx.Logs = []string{"log1", "log2"}
```

Comment:
What is all this doing? Some comments would help

Reply:
Done